src/commands/downlink/CorrectTime2000.ts

```diff
@@ -54,11 +47,13 @@ class CorrectTime2000 extends Command {
         super();
     }
 
+
     static readonly id = COMMAND_ID;
 
     static readonly directionType = DOWNLINK;
 
-    static readonly title = COMMAND_TITLE;
+    static readonly hasParameters = true;
+
```

Comment:
```suggestion
```

Reply:
in other files too, it's special separation or mistake?

Reply:
special separation